### DISCUSSION TITLE
Vorta on Pinephone

### DISCUSSION THREAD

**Discussion by jgarte** [borgbase/vorta#2079] [General]
Hi,

Has anyone tried using vorta with a Pinephone?

**Comment by m3nu** [comment]
Not a pinephone user. 😬

**Reply by jgarte** [reply]
[postmarketos](https://postmarketos.org/) (alpine linux variant distro) [has flatpak](https://pkgs.alpinelinux.org/packages?name=flatpak).

I'll try installing vorta that way and testing soonish.
